plugins/BEdita/Core/src/Shell/StreamsShell.php

```diff
@@ -77,4 +95,100 @@ public function removeOrphans()
         }
         $this->out(sprintf('%d stream(s) deleted', $count));
     }
+
+    /**
+     * Re-read streams metadata from file and update information in database.
+     *
+     * @return void
+     */
+    public function refreshMetadata()
+    {
+        $query = $this->Streams->find('all');
+        if ((bool)$this->param('force') === false) {
+            $query = $query->where(function (QueryExpression $exp): QueryExpression {
+                return $exp->or_(function (QueryExpression $exp): QueryExpression {
+                    return $exp
+                        ->isNull($this->Streams->aliasField('file_size'))
+                        ->isNull($this->Streams->aliasField('width'))
+                        ->isNull($this->Streams->aliasField('height'));
+                });
+            });
+        }
+
+        $count = $query->count();
+        $this->info(sprintf('Checking %d streams', $count));
```

Comment:
[minor] This number may change if new streams are saved while the script is running, or if streams are deleted, fixed by other means, …

I'd argue that this count may be useless, but if it is actually desired to give an idea of how much data is to be processed, at least I'd point out explicitly that this is an approximation:

```suggestion
        $this->info(sprintf('Approximately %d streams to check', $count));
```